pages/linux/sensible-editor.md

```diff
@@ -0,0 +1,19 @@
+# sensible-editor
+
+> Open the default editor.
+
+- Open a file in the default editor:
+
+`sensible-editor %{{file}}`
+
+- Open a file in the default editor, cursor at the end of the file:
+
+`sensible-editor + %{{file}}`
+
+- Open a file in the default editor, cursor at line 10:
```

Comment:
```suggestion
- Open a file in the default editor, with the cursor at line 10:
```

Also, is the cursor placed at the start or the end of the line?

Reply:
suggestion committed with edit, new line reads:
- Open a file in the default editor, with the cursor at the beginning of line 10: